src/mmw/js/src/core/opacityControl.js

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright (c) 2013, Jared Dominguez
+Copyright (c) 2013, LizardTech
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright notice, this
+  list of conditions and the following disclaimer in the documentation and/or
+  other materials provided with the distribution.
+
+* Neither the name of LizardTech nor the names of its
+  contributors may be used to endorse or promote products derived from
+  this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+        Leaflet.OpacityControls, a plugin for adjusting the opacity of a Leaflet map.
+        (c) 2013, Jared Dominguez
+        (c) 2013, LizardTech
+
+        https://github.com/lizardtechblog/Leaflet.OpacityControls
+
+*/
+
+'use strict';
+
+var L = require('leaflet'),
+    $ = require('jquery'),
+    Marionette = require('../../shim/backbone.marionette'),
+    sliderTmpl = require('./templates/opacitySlider.html');
+
+require('jquery-ui');
```

Comment:
`jquery-ui` must be added to `package.json`, `npm-shrinkwarp.json`, or `bundle.sh` before it can be used here.